### PR TITLE
Test high-order Argyris and HCT

### PR DESCRIPTION
**Pull request by pbrubeck**
# Description

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -82,6 +82,9 @@ jobs:
             --install defcon \
             --install gadopt \
             --install asQ \
+            --package-branch fiat pbrubeck/high-order-hct \
```
Comment (rckirby) [inline .github/workflows/build.yml:85]: Put these back to master/delete the branches

File: tests/regression/test_projection_zany.py [hunk 2]
```diff
@@ -13,10 +13,22 @@
 from firedrake import *
 
 
+relative_magnitudes = lambda x: np.array(x)[1:] / np.array(x)[:-1]
+convergence_orders = lambda x: -np.log2(relative_magnitudes(x))
+
+
+def jrc(a, b, n):
```
Comment (rckirby) [inline tests/regression/test_projection_zany.py:20]: Should we import this from FIAT?

**Review by rckirby — CHANGES_REQUESTED** [review]
Looks good.  We just need to put the packages back and make things still pass against master of FI(n)AT + tsfc.

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -82,6 +82,9 @@ jobs:
...
+            --package-branch finat pbrubeck/high-order-hct \
+            --package-branch tsfc pbrubeck/argyris-variant \
```
Comment (pbrubeck) [inline .github/workflows/build.yml:87]:
```suggestion
```

**Review by rckirby — APPROVED** [review]

**Pull request merged**